### ISSUE TITLE
How can we add a new field?

### ISSUE THREAD

**Issue by mohammad79-ch** [darryldecode/laravelshoppingcart#326]
How can we add a new field in the shopping cart?

**Comment by Blum** [comment]
Use "attributes" in each item added to the cart, you can add there whatever you want.

**Comment by mohammad79-ch** [comment]
Yeah right , and how can the total amount be updated after applying the discount?
